00-Raster-Structure.Rmd

```diff
@@ -83,9 +81,7 @@ NEON Data Skills: Image Raster Data in R - An Intro</a>
 </div>
 
 ## About Raster Data
-Raster or "gridded" data are saved on a regular grid which is rendered on a map
-as pixels. Each pixel contains a value that represents an area on the Earth's 
-surface.
+Raster or "gridded" data are stored as a grid of values which are rendered on a map as pixels. Each pixel value represents an area on the Earth's surface.
```

Comment:
hi @brymz . I should document this online however we stick to a 80 character width limit on our md / Rmd files like data and software carpentry do. If you don't have time to adjust this, i can easily do it after I merge. It looks like this is all on one line.